library/src/main/java/io/ona/kujaku/views/KujakuMapView.java

```diff
@@ -893,17 +911,18 @@ public void focusOnUserLocation(boolean focusOnMyLocation) {
     @Override
     public void focusOnUserLocation(boolean focusOnMyLocation, Float radius) {
         if (focusOnMyLocation) {
-            isMapScrolled = false;
+            //isMapScrolled = false;
```

Comment:
Why have you commented this out???

Reply:
because isMapScrolled is not use anymore and replaced by the 2 variables `updateUserLocationOnMap` & `updateCameraUserLocationOnMap`

Reply:
Cool, and the code still works. However, I can't see where `updateUserLocationOnMap` becomes false at any point now. It is initially false, but you switched out it's at the if with `updateCameraUserLocationOnMap` and added an extra if for changing the marker on the map

Reply:
As soon as you want to see the user location on map, the variable updateUserLocationOnMap becomes true. When you open the map you always want to see your real position on the map, right ? it was not possible before

Reply:
Maybe we need a method that remove the user location on the map

Reply:
I probably don't get you because the MapView does not use start the location services by default and therefore it never has the real position and the marker will also not be showing on the map by default. Could you kindly explain what you mean?

Reply:
That is why the flag is false at first (No location on the map) and becomes true as soon as you use the method "focusOnUserLocation(true)"